ajax/libs/egjs-agent/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "egjs-agent",
+  "filename": "agent.min.js",
+  "version": "2.1.1",
+  "description": "Extracts browser and operating system information from the user agent string.",
+  "author": "NAVER Corp.",
+  "license": "MIT",
+  "keywords": [
+    "egjs",
+    "agent"
```

Comment:
Where are those keywords from? The agent is meaningless here, use `user agent` or `user`, `agent` instead, maybe even `browser` and `operating system`

Reply:
Keyword from GitHub repo blue part (topics: categorize the repository)
![2017-09-11 11 00 33](https://user-images.githubusercontent.com/8349587/30256911-bc909332-96e0-11e7-8ee1-ba7ef4e3872f.png)
Thank you.

Reply:
@happyhj @sculove could you give us some keywords about this lib?
It's will be useful to us, and let this lib better to be search.
Thank you.

Reply:
I've mentioned the keywords we can use here, just add them please.